src/components/ClusterNodes/ClusterTerminal.tsx

```diff
@@ -951,6 +976,14 @@ export default function ClusterTerminal({
                 buttonSelectionState: manifestButtonState,
                 setManifestButtonState: setManifestButtonState,
             },
+            ...(showManagedFieldsCheckbox
+                ? [
+                      {
+                          type: TerminalWrapperType.CUSTOM_COMPONENT,
+                          customComponent: renderHideManagedFields,
+                      },
+                  ]
+                : []),
```

Comment:
```suggestion
            ...(showManagedFieldsCheckbox
                && [
                     {
                          type: TerminalWrapperType.CUSTOM_COMPONENT,
                          customComponent: renderHideManagedFields,
                      },
                  ]
                ),
```

Reply:
Would cause issue: not iterable in case of false